presto-product-tests/conf/tempto/tempto-configuration.yaml

```diff
@@ -3,3 +3,23 @@ databases:
     host: hadoop-master
   presto:
     host: presto-master
+
+  prestoConfigUserAlice:
+    host: localhost
+    server_address: ${databases.presto.host}:8080
+    cli_server_address: ${databases.presto.host}:8080
+    jdbc_driver_class: com.facebook.presto.jdbc.PrestoDriver
+    jdbc_url: jdbc:presto://${databases.presto.server_address}/hive/${databases.hive.schema}
+    jdbc_user: alice
+    jdbc_password: na
+    jdbc_pooling: false
+
+  prestoConfigUserBob:
+    host: localhost
+    server_address: ${databases.presto.host}:8080
+    cli_server_address: ${databases.presto.host}:8080
+    jdbc_driver_class: com.facebook.presto.jdbc.PrestoDriver
+    jdbc_url: jdbc:presto://${databases.presto.server_address}/hive/${databases.hive.schema}
+    jdbc_user: bob
+    jdbc_password: na
+    jdbc_pooling: false
```

Comment:
conf/tempto/tempto-configuration.yaml is used by run_on_docker, and the other one I think was used when we used to run product tests using [tempto runner](https://github.com/prestodb/tempto#using-tempto-runner)